src/game.cpp

```diff
@@ -1789,10 +1789,83 @@ void game::update_weather()
     }
 }
 
+int game::get_heat_radiation( const tripoint &location, bool direct )
+{
+    // Direct heat from fire sources
+    // Cache fires to avoid scanning the map around us bp times
+    // Stored as intensity-distance pairs
+    int temp_mod = 0;
+    std::vector<std::pair<int, int>> fires;
+    fires.reserve( 13 * 13 );
+    int best_fire = 0;
+    for( const tripoint &dest : g->m.points_in_radius( location, 6 ) ) {
```

Comment:
Whether function is moved from `class game` to be a free function, or remains as is, can shorten all instances of `g->m.__func__` to just `m.__func__`
Other lines impacted:
* 1804
* 1807
* 1810

Function is only called currently as `g->get_heat_radiation` so it already has access to the global-g's map.